model/src/test/resources/oracle/kubernetes/weblogic/domain/v2/domain-sample-2.yaml

```diff
@@ -77,6 +77,9 @@ spec:
           periodSeconds: 20
         nodeSelector:
           os: linux
+        serviceLabels:
```

Comment:
Yes, single letter names work, but they require the next developer to read a lot more than necessary. Things like "label1" or "name1" and "value1" are a lot easier to understand.

Reply:
i have replaced the single character label and annotation names as requested